industrial_ci/src/docker.sh

```diff
@@ -163,7 +163,7 @@ function ici_prepare_docker_image() {
 #   (None)
 
 function ici_build_default_docker_image() {
-  export DOCKER_IMAGE="industrial-ci/$OS_CODE_NAME"
+  export DOCKER_IMAGE="industrial-ci/${OS_CODE_NAME:-${DOCKER_BASE_IMAGE##*:}}"
```

Comment:
Here we're assuming `DOCKER_BASE_IMAGE` is already set in a script previously run (I think `env.sh` in this case). In other words `docker.sh` is dependent on another script(s). Would be great if we can leave inline comment for that.